payment-instruments/trustly/features/optional/payment-link.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Payment Link
+estimated_read: 5
+description: |
+  Sending the payment via mail or SMS.
```

Comment:
When the `description` (or other front matter properties) don't need to span multiple lines, I think it's cleaner to drop the `|` and just make it a one-liner.

```suggestion
description: Sending the payment via mail or SMS.
```